webbpsf/roman.py

```diff
@@ -911,7 +911,7 @@ class CGI(RomanInstrument):
         'DISKSPC_F721': ('IMAGER', 'F721', 'DISKSPC', 'DISKSPC_F721_ANNULUS', 'LS30D88')}
 
     def __init__(self, mode=None, pixelscale=None, fov_arcsec=None, apply_static_opd=False):
-        super(CGI, self).__init__("CGI", pixelscale=pixelscale)
+        super(Coronagraph, self).__init__("CGI", pixelscale=pixelscale)
```

Comment:
This could be just `super().__init__` now. The older syntax is leftover from Py2.7. [optional change, not a big deal either way]

Reply:
I made this change throughout the file for consistency's sake.